go.mod

```diff
@@ -6,6 +6,7 @@ toolchain go1.21.3
 
 // v1.5.0 has broken uint parsing, use a commit with the fix until the
 // the maintainers merge the PR: https://github.com/spf13/cast/pull/144
+replace github.com/oasisprotocol/oasis-sdk/client-sdk/go => ../oasis-sdk/client-sdk/go
```

Comment:
This seems wrong.

Reply:
Yeah, that's the debug commit. Will drop it when merging.